LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2016-2017 Crown copyright (Ministry of Justice)
```

Comment:
the year should start with the current year and then if any subsequent development takes place in subsequent years then include thm

Reply:
will update in follow up